README.md

```diff
@@ -89,15 +90,16 @@ After that call the `install.sh script.
 
 ```bash
 cp /data/dbus-opendtu/config.ini /data/dbus-opendtu/config.backup
-wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/heads/main.zip
-unzip main.zip "dbus-opendtu-main/*" -d /data
+wget -O main.zip https://github.com/henne49/dbus-opendtu/archive/refs/tags/$(curl -s https://api.github.com/repos/henne49/dbus-opendtu/releases/latest | grep "tag_name" | cut -d '"' -f 4).zip
+mkdir temp_dir
+unzip main.zip -d temp_dir && find temp_dir -mindepth 1 -maxdepth 1 -type d -exec mv {} /data/dbus-opendtu-main/ \; && rmdir temp_dir
```

Comment:
I really appreciate the effort and intention to provide a better user experience. However, I think the current approach could be improved in terms of readability and maintainability.

Here's a suggestion that I believe is more straightforward and easier to understand:


```suggestion
set -e
unzip main.zip -d temp_dir
mv temp_dir/* /data/dbus-opendtu-main/
rmdir temp_dir
```

The -e option ensures that the script exits immediately if any command fails, which improves robustness and error handling. I hope this suggestion is helpful!